src/Map/src/Bridge/Google/src/Option/GestureHandling.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map\Bridge\Google\Option;
+
+/**
+ * This setting controls how the API handles gestures on the map.
+ *
+ * @see https://developers.google.com/maps/documentation/javascript/reference/map#MapOptions.gestureHandling
+ *
+ * @author Hugo Alliaume <hugo@alliau.me>
+ */
+enum GestureHandling: string
+{
+    /**
+     * Scroll events and one-finger touch gestures scroll the page, and do not zoom or pan the map.
+     * Two-finger touch gestures pan and zoom the map.
+     * Scroll events with a ctrl key or ⌘ key pressed zoom the map.
+     * In this mode the map cooperates with the page.
+     */
+    case Cooperative = 'cooperative';
+
+    /**
+     * All touch gestures and scroll events pan or zoom the map.
+     */
+    case Greedy = 'greedy';
+
+    /**
+     * The map cannot be panned or zoomed by user gestures.
+     */
+    case None = 'none';
+
+    /**
+     * Gesture handling is either cooperative or greedy, depending on whether the page is scrollable or in an iframe.
+     */
+    case Auto = 'auto';
```

Comment:
That's why i personnaly would not have used Enums here :)

Reply:
Not sure to understand why

Reply:
Enum should be "partition of some set".. here you can be both auto and another one for instance... and the value can change.

Reply:
> you can be both auto and another one for instance

I'm only seeing 4 allowed values for `gestureHandling` option, and only one of them is allowed at the same time:
<img width="1013" alt="image" src="https://github.com/user-attachments/assets/1a72c321-ce1d-4ef7-b3fa-548314a2c063">
https://developers.google.com/maps/documentation/javascript/reference/map#MapOptions.gestureHandling